<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="appBar">
      <property name="minimumSize">
       <size>
        <width>661</width>
        <height>61</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FF2471FF;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">Color:#FFFFFF</string>
       </property>
       <property name="text">
        <string>Detailed Task</string>
       </property>
      </widget>
      <widget class="QPushButton" name="backButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FFFFFF;
    color: black;
    border-radius: 10px;
    font-size: 12px;
    font-weight: bold;
    padding: 8px 15px;
}

QPushButton:hover {
    background-color: #164499;
}

QPushButton:pressed {
    background-color: #0E326F;
}
</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="taskInput">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 2px solid #2471FF;
    border-radius: 10px;
    padding: 6px;
    font-size: 14px;
    color: #000000;
}

QLineEdit:focus {
    border: 2px solid #164499;
    background-color: #F5F8FF;
}
</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Enter Your Task</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="timeInput">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 2px solid #2471FF;
    border-radius: 10px;
    padding: 6px;
    font-size: 14px;
    color: #000000;
}

QLineEdit:focus {
    border: 2px solid #164499;
    background-color: #F5F8FF;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Enter The Time</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="effortCombo">
      <property name="styleSheet">
       <string notr="true">QComboBox {
    background-color: #FFFFFF;
    border: 2px solid #2471FF;
    border-radius: 10px;
    padding: 6px;
    font-size: 14px;
    color: #000000;
}

QComboBox:focus {
    border: 2px solid #164499;
    background-color: #F5F8FF;
}



QComboBox QAbstractItemView {
    border: 1px solid #2471FF;
    border-radius: 6px;
    background-color: #FFFFFF;
    selection-background-color: #2471FF;
    selection-color: #FFFFFF;
}
</string>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Effort level</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hard</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Easy</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addTaskButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #2471FF;
    color: white;
    border-radius: 10px;
    font-size: 15px;
    font-weight: bold;
    padding: 8px 15px;
}

QPushButton:hover {
    background-color: #164499;
}

QPushButton:pressed {
    background-color: #0E326F;
}
</string>
      </property>
      <property name="text">
       <string>Add Task</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>659</width>
         <height>236</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="task_container" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
